meta-intel-edison-distro/recipes-core/images/edison-image.bb

```diff
@@ -76,6 +76,10 @@ IMAGE_INSTALL:append = " libiio"
 IMAGE_INSTALL:append = " libiio-iiod"
 IMAGE_INSTALL:append = " libiio-python3"
 
+# gpiod replaces the former MRAA and UPM to access sensor/actuator drivers in the kernel
```

Comment:
Wondering if you can pack libgpiod 2.0+ into experimental branch. It includes support of GPIO Character Device v2 ABI, which has a lot of nice features.

Reply:
I put that on the road map for Kirkstone as is not yet even a -rc.